Apps/W1/HybridBaseDeployment/app/src/codeunits/HybridCloudManagement.Codeunit.al

```diff
@@ -2223,6 +2223,23 @@ codeunit 4001 "Hybrid Cloud Management"
         RecordLink.SetRange("To User ID", OldUserName);
         RecordLink.ModifyAll("To User ID", NewUserName);
     end;
+    
+    local procedure ClearCompanyMigrationValidation(MigrationType: Text[250])
+    var
+        MigrationValidationError: Record "Migration Validation Error";
+        CompanyValidationProgress: Record "Company Validation Progress";
+    begin
+        if MigrationType <> '' then
+            MigrationValidationError.SetRange("Migration Type", MigrationType);
+
+        MigrationValidationError.SetRange("Company Name", CompanyName());
+        if not MigrationValidationError.IsEmpty() then
+            MigrationValidationError.DeleteAll();
```

Comment:
In general it is better to soft delete, e.g. set the property dismissed to true. Then in the UI you can choose if you want to see the old data or not.